controlnet.py

```diff
@@ -78,6 +67,25 @@ def __init__(self, image_path, device, fp16, vram_O, type='scribble', hed=False)
         self.max_step = int(self.num_train_timesteps * 0.98)
         self.alphas = self.scheduler.alphas_cumprod.to(self.device)  # for convenience
 
+        # Prepare image
+        assert guidance_image_path, "Must provide guidance_image_path for ControlNet guidance!"
+        height, width = 512, 512
+        self.image = load_image(guidance_image_path)
+        hed = HEDdetector.from_pretrained('lllyasviel/ControlNet')
```

Comment:
why do hed when our guidance image is a scribble? I thought hed was used to generate a scribble-like image